packages/vite/src/node/plugins/css.ts

```diff
@@ -998,7 +998,9 @@ const scss: SassStylePreprocessor = async (
   const internalImporter: Sass.Importer = (url, importer, done) => {
     resolvers.sass(url, importer).then((resolved) => {
       if (resolved) {
-        rebaseUrls(resolved, options.filename, options.alias).then(done)
+        rebaseUrls(resolved, options.filename, options.alias)
+          .then(done)
+          .catch(done)
```

Comment:
We could merge the calls
```suggestion
        rebaseUrls(resolved, options.filename, options.alias).finally(done)
```

But shouldn't we log this error? You mention it was reported maybe I'm missing something.

> Would it make sense to add a case where a file is edited to add a missing import in packages/playground/css/__tests__/css.spec.ts?

I think this is a good idea 👍🏼, with a comment so the intention is clear

Reply:
@patak-js @Shinigami92 It's important that the callback receives either the result or the error, so using `finally` is not an option in this case.

When the callback receives an error, it will propagate it and Vite will display it in the terminal and overlay, that's what I meant by "reported" 😃

Reply:
Ah, I see